res/database/Default/user/jorgaeff.xml

```diff
@@ -82,9 +94,11 @@
 			<availability script="TIME_YEAR=1985" />
 			<title>
 				<de>APD mit historischem Wahlergebnis</de>
+				<en>APD with historical election result</en>
 			</title>
 			<description>
 				<de>Bei der Landtagswahl in Nordrhein-Westfalen erzielt die Arbeiterpartei Deutschland (APD) unter Ministerpräsident Hannes Reif mit 52,1% der Stimmen das beste Wahlergebnis ihrer Geschichte in diesem Bundesland.</de>
+				<en>In the state elections in North Rhine-Westphalia, the Arbeiterpartei Deutschland (APD) headed by Prime Minister Hannes Reif achieves the best election result in its history in this state, with 52.1% of the vote.</en>
```

Comment:
of the votes?

Reply:
"The party got/took 25 percent of the vote." https://dictionary.cambridge.org/dictionary/english/vote